client/src/Try/Container.purs

```diff
@@ -505,13 +498,13 @@ toAnnotation markerType { position, message } =
     , text: message
     }
 
-withSession :: String -> Aff { sourceFile :: Maybe SourceFile, code :: String }
-withSession sessionId = do
-  state <- H.liftEffect $ tryRetrieveSession sessionId
-  githubId <- H.liftEffect $ getQueryStringMaybe "github"
+withSession :: Aff { sourceFile :: Maybe SourceFile, code :: String }
+withSession = do
```

Comment:
Also maybe `withSession` no longer makes sense, now that this doesn't take a `sessionId`

Reply:
What are your thoughts on what this could be renamed to then?

Reply:
Couldn't think of anything better 😅